src/components/workers-page/management-page.component.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { Divider, Tab } from "@material-ui/core";
+import TabContext from "@material-ui/lab/TabContext";
+import TabList from "@material-ui/lab/TabList";
+import TabPanel from "@material-ui/lab/TabPanel";
+import ShiftTab from "./shifts-tab/shifts-tab.component";
+import ConstraintTab from "./constraints-tab/constraints-tab.component";
+import WorkersTab from "./workers-tab/workers-tab.component";
+import { ImportButtonsComponent } from "../schedule-page/import-buttons/import-buttons.component";
+
+interface TabData {
+  label: string;
+  component: JSX.Element;
+}
+
+export default function ManagementPage(): JSX.Element {
+  const tabs: TabData[] = [
+    { label: "Pracownicy", component: <WorkersTab /> },
```

Comment:
Wczoraj z @VVlovsky implementowaliśmy ten komponent, i zrobiliśmy tak, że on jako argument przyjmuje słownik w którym kluczem jest nazwa zakładki, a wartością - konstruktor komponentu który musi być utworzony. 
I teraz powstaje pytanie: 
czy zostawiamy słownik czy listę obiektów?
Osobiście ja nie widzę większej różnicy a zmiana jest niewielka, więc myślę, że możemy zrobić to zarówno tutaj jak i w tym PR, co będzie wrzucać @VVlovsky

Reply:
Dokładnie ten komponent? OO Cieżko stwierdzić nie wdziąc tego kodu. Może faktycznie jest lepiej tak jak piszesz. Proponuję domergować to zrobione w taki sposób i jeśli @VVlovsky uzna, że jego metoda jest lepsza to ma moje błogosławieństwo do zmieniania tego.